src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        String randomColor = supplier.getRandomColor();
+
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
```

Comment:
lets make ColorSupplier and Random class-level variables - to avoid creating those objects each time this method is invoked

lets add constant for 100 - don't use magic numbers